doc/user/backup_options.md

```diff
@@ -0,0 +1,26 @@
+# Backup Options Setup Guide
+
+In etcd operator, we provide the following options to save cluster backups to:
+- Persistent Volume (PV) on GCE or AWS
+- S3 bucket on AWS
+
+## PV on GCE
+
+By default, operator supports saving backup to PV on GCE.
+This is done by passing flag `--pv-provisioner=kubernetes.io/gce-pd` to operator, which is also the default value.
+This is essentially saving backups to an instance of GCE PD.
+
+## PV on AWS
+
+If running on AWS Kubernetes, it needs to pass flag `--pv-provisioner=kubernetes.io/aws-ebs` to operator.
```

Comment:
Instead of:
it needs to pass flag `--pv-provisioner=kubernetes.io/aws-ebs` to operator
maybe:
pass the flag `--pv-provisioner=kubernetes.io/aws-ebs` to the operator